.github/workflows/gh-pages.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         run: sudo apt-get update
 
       - name: Setup | System
-        run: sudo apt-get install doxygen python3-sphinx opam libgmp-dev ninja-build
+        run: sudo apt-get install doxygen python3-sphinx opam libgmp-dev ninja-build nodejs
```

Comment:
~question: To follow along, I had to run `sudo apt install jsdoc-toolkit` as well. Is this necessary?~

Nevermind. There is a `npm install jsdoc` later.